### PR TITLE
Add support for task inputs

### PR DESCRIPTION
**Pull request by rgoomar**

### CODE REVIEW

**Comment by david-cako** [comment]
Sweet!  I'll give this a look sometime this week.

File: toggl_hammer/toggl_hammer.py [hunk 1]
```diff
@@ -91,21 +91,29 @@ def entry_prompt(self, date_index):
             print("[{0}] {1}".format(i, project[0]))
         proj_index = input("choose project: ")
         proj_index = int(proj_index)
+        proj_id = self.projects[proj_index][1]
+        task_list = requests.get('https://www.toggl.com/api/v8/projects/' + str(proj_id) + '/tasks', auth=(API_KEY, 'api_token')).json()
+        for i, task in enumerate(task_list):
```
Comment (david-cako) [inline toggl_hammer/toggl_hammer.py:96]: Does not account for `task_list == None`, which is what you get if there are no tasks.  Maybe only if your account doesn't support them?  Not sure.

**Comment by david-cako** [comment]
Ok, so I unfortunately do not have access to the "tasks" feature because I use toggl's free tier, however I did find that your branch does not account for the case where no tasks are returned.  I made some changes that I can't test.

Give https://github.com/david-cako/toggl-hammer/commit/4770343aa6c93f1fe4ddbdbc9e7bbc87a29de44a a try and let me know if it works.

**Comment by rgoomar** [comment]
Tested it out. It works well. You can use that commit / branch instead of this PR.

**Pull request merged**